packages/react-components/react-tree/stories/Tree/TreeWithoutInlineStyle.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {
+  Tree,
+  TreeItem as BaseTreeItem,
+  TreeItemLayout,
+  treeItemLevelToken,
+  TreeItemProps,
+  useTreeContext_unstable,
+} from '@fluentui/react-tree';
+import { makeStyles, mergeClasses } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  level1: {
```

Comment:
What is we just hard coded like 10 level support and if any further levels are needed, we just show users how to add new levels... if users are going to override with their own level styling anyway what's the point?

Reply:
For users with lots of custom levels we can instruct them how to use the inline style method (that is currently internal) to handle that.

This way for users that won't care about more than 10 levels, it's much easier to override the styles

Reply:
I like the idea, going with static first and documenting the downsides and how to use inline sounds like a good alternative 👍🏻